File: tt-train/sources/ttml/metal/ops/softmax/softmax.cpp
// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
//
// SPDX-License-Identifier: Apache-2.0

#include "softmax.hpp"

namespace ttml::metal {

ttnn::Tensor softmax(const ttnn::Tensor& input_tensor, int32_t dim) {
    throw std::runtime_error("softmax operation has been removed");
}

}  // namespace ttml::metal

File: tests/ttnn/unit_tests/operations/softmax/softmax.py
# SPDX-FileCopyrightText: (c) 2025 Tenstorrent Inc.
# SPDX-License-Identifier: Apache-2.0

"""
Re-export softmax from the operation package for test convenience.

Stage tests import as: from .softmax import softmax
"""

from ttnn.operations.softmax import softmax

__all__ = ["softmax"]
